account_multicurrency_revaluation_type/wizard/wizard_currency_revaluation.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class WizardCurrencyRevaluationType(models.TransientModel):
+
+    _inherit = 'wizard.currency.revaluation'
+
+    # Todo get in count direct/indirect rate
```

Comment:
Is this TODO still relevant? If it's optional, can it be added to the ROADMAP section?

Reply:
yes, but it should be done in other module